software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -80,9 +84,22 @@ public void launch() {
 
     @Override
     public boolean isRunning() {
-        int exitCode = executeCommandInTask(
-                getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_COMMAND),
-                getEntity().getConfig(VanillaWindowsProcess.CHECK_RUNNING_POWERSHELL_COMMAND), "is-running-command");
+        int exitCode = 1;
```

Comment:
Very minor (for next time). I wouldn't bother giving this an initial value. It will always get a value inside the try-block, and the catch block will always either return or throw. So the initial value of `1` will always get overwritten. Better to be explicit about that by not giving it any initial value.